Models/Measurement.cs

```diff
@@ -447,6 +452,9 @@ public void reload(Spectrometer spec)
         dark = spec.dark;
         postProcess();
 
+        roiStart = spec is null ? 0 : (uint)spec.eeprom.ROIHorizStart;
+        roiEnd = spec is null ? 1024 : (uint)spec.eeprom.ROIHorizEnd;
```

Comment:
I guess I'd prefer pixels() to 1024 if possible?

Reply:
here spec is the spectrometer, so if that's null and this is being called that's a problem, but I do agree the previous pixel number makes more sense